tests/constraints.txt

```diff
@@ -21,5 +21,5 @@ pytest-dependency==0.5.1
 pytest-html==3.2.0
 pytest-timeout==2.1.0
 jax<=0.4.14
```

Comment:
```suggestion
jax<=0.4.18
```

Reply:
@ilya-lavrenov, please help to commit this change.
0.4.15 was yanked and not it looks better version to update